src/hmmdwrkr.c

```diff
@@ -191,6 +211,280 @@ worker_process(ESL_GETOPTS *go)
   return;
 }
 
+static int
+do_sq_by_sequences(ESL_GENCODE *gcode, ESL_GENCODE_WORKSTATE *wrk, ESL_SQ *sq)
+{
+      if (wrk->do_watson) {
+	esl_gencode_ProcessStart(gcode, wrk, sq);
+	esl_gencode_ProcessPiece(gcode, wrk, sq);
+	esl_gencode_ProcessEnd(wrk, sq);
+      }
+
+      if (wrk->do_crick) {
+	esl_sq_ReverseComplement(sq);
+	esl_gencode_ProcessStart(gcode, wrk, sq);
+	esl_gencode_ProcessPiece(gcode, wrk, sq);
+	esl_gencode_ProcessEnd(wrk, sq);
+      }
+
+  return eslOK;
+}
+
+static void 
+process_TranslatedSearchCmd(HMMD_COMMAND *cmd, WORKER_ENV *env, QUEUE_DATA *query)
+{ 
+  int              i;
+  int              cnt;
+  int              limit;
+  int              status;
+  int              blk_size;
+  WORKER_INFO     *info       = NULL;
+  ESL_ALPHABET    *abc;
+  ESL_STOPWATCH   *w;
+  ESL_THREADS     *threadObj  = NULL;
+  pthread_mutex_t  inx_mutex;
+  int              current_index;
+  time_t           date;
+  char             timestamp[32];
+
+  int              k;
+  ESL_ALPHABET     *abcDNA = NULL;       /* DNA sequence alphabet                               */
+  ESL_GENCODE      *gcode       = NULL;
+  ESL_GENCODE_WORKSTATE *wrk    = NULL;
+  ESL_SQ           *qsqDNATxt = NULL;    /* DNA query sequence that will be in text mode for printing */
+  ESL_SQ           *qsq      = NULL;		 /* query sequence */
+  P7_TOPHITS       *tophits_accumulator = NULL; /* to hold the top hits information from all 6 frame translations */
+  P7_PIPELINE      *pipelinehits_accumulator = NULL; /* to hold the pipeline hit information from all 6 frame translations */
+  RANGE_LIST       *range_list;  /* (optional) list of ranges searched within the seqdb */
+ 
+
+  if (query->cmd_type == HMMD_CMD_SEARCH) {
+    LOG_FATAL_MSG("ERROR: hmmsearcht option is not implemented", EPERM);
+    return;
+  }
+
+  w = esl_stopwatch_Create();
+  abc = esl_alphabet_Create(eslAMINO);
+  abcDNA = esl_alphabet_Create(eslDNA); 
+
+  if (pthread_mutex_init(&inx_mutex, NULL) != 0) p7_Fail("mutex init failed");
+  ESL_ALLOC(info, sizeof(*info) * (env->ncpus));
+
+  /* Log the current time (at search start) */
+  date = time(NULL);
+  ctime_r(&date, timestamp);
+  printf("\n%s", timestamp);	/* note that ctime_r() leaves \n on end of timestamp  */
```

Comment:
Where is this printing to? This is a daemon worker; don't rely on it having `stdout`.